src/main/java/com/google/cloud/tools/appengine/operations/AppYamlProjectStaging.java

```diff
@@ -108,6 +122,15 @@ void stageStandardArchive(AppYamlProjectStageConfiguration config)
     copyArtifactJarClasspath(config, copyService);
   }
 
+  @VisibleForTesting
+  void stageStandardBinary(AppYamlProjectStageConfiguration config)
+      throws IOException, AppEngineException {
+    CopyService copyService = new CopyService();
+    copyExtraFiles(config, copyService);
+    copyAppEngineContext(config, copyService);
+    copyArtifact(config, copyService);
```

Comment:
Also update the comment on [this line](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/df426fd57c7568659d702992d9eaf50503bdd37e/src/main/java/com/google/cloud/tools/appengine/operations/AppYamlProjectStaging.java#L201) that it's not only JAR or WAR?